tests/test_cobalt_activity.py

```
@@ -0,0 +1,162 @@
+    co60_activity = activities.get('Co060', [0])[0]  # Note: returned keys are capitalized
+    if isinstance(co60_activity, np.ndarray):
+        co60_activity = float(co60_activity.flat[0])
+    else:
+        co60_activity = float(co60_activity)
+    
+    # Expected value from reference calculation
+    expected_activity = 233.89  # Bq
+    tolerance = 0.01  # 1%
+    
+    # Print result for user visibility
+    print(f"\n✓ F4Epurity calculation complete:")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)